### PR TITLE
Add log browsing function

### PR DESCRIPTION
**Pull request by blue1st**

### CODE REVIEW

File: lib/digdag.go [hunk 1]
```diff
@@ -106,3 +106,20 @@ func (c *Client) doReq(method, spath string, params, res interface{}) error {
 
 	return req.Body.FromJsonTo(&res)
 }
+
+func (c *Client) doReqRaw(method, spath string, params interface{}) (string, error) {
```
Comment (szyn) [inline lib/digdag.go]: Please fix this method name to `doRawReq`.

File: lib/attempt.go [hunk 2]
```diff
@@ -69,6 +73,19 @@ type Task struct {
 	IsGroup      bool          `json:"isGroup"`
 }
 
+type logfiles struct {
```
Comment (szyn) [inline lib/attempt.go]: `lib/attempt.go` should related attempt. (e.g. spath contains `/attempts`)
So if you want to these changes, could you create a new file (`lib/log.go`) and put it in there?

**Review by szyn — CHANGES_REQUESTED** [review]
Thanks for the PR! Please check the comments.

**Comment by blue1st** [comment]
Thank you for the review! Fixed the problem.

**Review by szyn — APPROVED** [review]
Looks good to me! That's nice features👍  
Thank you.

**Pull request merged**